src/dmd/tocvdebug.d

```diff
@@ -54,8 +51,6 @@ import dmd.backend.oper;
 import dmd.backend.ty;
 import dmd.backend.type;
 
-extern (C++):
```

Comment:
Is this related?

Reply:
This is a dmd-only backend module, I can't see any reason why these must be exported to C++.

Reply:
Okay, I just wondered if removing it was needed for cross compilation

Reply:
No I was just cleaning out old cruft.